test/pt2_to_circle_test/builder.py

```diff
@@ -92,17 +97,25 @@ def _run(self, without_pt2=False):
             # torch.nn.Module --> ExportedProgram --> pt2 ----- (ExportedProgram) ------- > circle
             #                                       (--> load_from_pt2_file -->)
             convert_nnmodule_to_circle(
-                self.nnmodule, self.example_inputs, circle_model_path
+                self.nnmodule,
+                self.example_inputs,
+                circle_model_path,
+                dynamic_shapes=dynamic_shapes,
             )
         else:
             # torch.nn.Module --> ExportedProgram ----------------------------------------> circle
-            convert_nnmodule_to_pt2(self.nnmodule, self.example_inputs, pt2_model_path)
+            convert_nnmodule_to_pt2(
+                self.nnmodule,
+                self.example_inputs,
+                pt2_model_path,
+                dynamic_shapes=dynamic_shapes,
+            )
             convert_pt2_to_circle(pt2_model_path, circle_model_path)
 
         verify_circle(circle_model_path, opt_circle_model_path)
 
         torch_result = infer_nnmodule(self.nnmodule, self.example_inputs)
-        USE_ONERT = os.environ.get("CCEX_RUNTIME") == "onert"
+        USE_ONERT = os.environ.get("CCEX_RUNTIME") == "onert" or dynamic
```

Comment:
May I ask what `CCEX_RUNTIME` is?

Reply:
You can refer to "Runtime Opitons" section of README.md. It's runtime thing.